microsoft/onefuzz
# Rename supervisor to agent

## Summary of the Pull Request

_What is this about?_

Pat 2 of 2 PRs for renaming agent to task.

## PR Checklist
* [ ] Closes #1956

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

- Ran check-pr
- No errors,
- Crashes reported
- Logs created
- Tasks are run
